### ISSUE TITLE
Execute actual queries in the same order as actions were performed

### ISSUE THREAD

**Issue by bocharsky-bw** [TablePlus/TablePlus#1166]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0): MySQL 

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81): 222

3. The steps to reproduce this issue:

It feels like DataPlus trying to replay SQL queries in a wrong order they were created after you did some changes and press Cmd + S to actually execute queries.

For example, I have a few tables:
- `orders`
- `payments`

If I delete a row from `payments` first, then delete a row from `orders` next, and when I press Cmd + S to actually execute queries - TablePlus show me them in a different order they were performed (probably it shows alphabetically them), i.e. it tries to delete from `orders` first, and then from `payments`, i.e. queries order is:
```sql
DELETE FROM `orders`
DELETE FROM `payments`
```

But this way it might fail if you have some constraints, i.e. when you need to remove exactly payment first and only then order. But won't it be better to execute actual queries in the same order we perform actions in the table view?

**Comment by jpjonte** [comment]
This feature would be very useful. It would also be great to be able to reorder the queries manually.

**Comment by huyphams** [comment]
Hi, I think this is a misunderstanding, TablePlus has no control over this. It sends the query to the database server and receives the result.
You might want to put the semicolons at the end of the query if you want TablePlus to execute them one by one.

**Comment by jpjonte** [comment]
Hi @huyphams, this refers to queries that TablePlus generates when you delete rows in the GUI and press CMD + S to send them to the server, not queries written by hand.

Currently it seems to order the generated queries in a fixed way, instead of ordering them by the order the actions were made in the GUI.

**Comment by bocharsky-bw** [comment]
Yeah, this is still relevant, thanks for explaining it @jpjonte !

@huyphams I have a little example in the PR's description btw :)
